include/crm/lrmd.h

```diff
@@ -409,6 +410,16 @@ typedef struct lrmd_api_operations_s {
                          const char *class,
                          const char *provider,
                          const char *agent, char **output, enum lrmd_call_options options);
+    /*!
+     * \brief Get the resource agent version of a resource agent.
+     * 
+     * \retval version on success
+     * \retval value "0.1" on failure
+     */
+    const char* (*get_version) (lrmd_t * lrmd,
+                                const char *class,
+                                const char *provider,
+                                const char *agent, enum lrmd_call_options options);
```

Comment:
I think I'd prefer to do this as an exposed library function (like, for example, lrmd_list_freeall()) rather than an API interface. It's more of a convenience function for using the API than a direct part of it, and it should never be implemented differently for different types of connections.